lib/oga/xml/element.rb

```diff
@@ -113,6 +117,8 @@ def set(name, value)
         if found
           found.value = value
         else
+          name = name.to_s if name.is_a?(Symbol)
+
```

Comment:
Same here. The code below should just use `split_name` similar to the `get` method, then this check is not needed.